tutorials/applications/QAA to solve a MWIS problem.ipynb

```diff
@@ -0,0 +1,522 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's attempt a simple minimization procedure to find the optimal positions of the atoms in the Register that replicate best the off-diagonal terms of $Q$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def evaluate_mapping(\n",
+    "    new_coords: np.ndarray, Q: np.ndarray, device: pulser.devices.Device\n",
+    "):\n",
+    "    \"\"\"Cost function to minimize. Ideally, the pairwise distances are conserved.\"\"\"\n",
+    "    new_coords = np.reshape(new_coords, (len(Q), 2))\n",
+    "    # computing the matrix of the distances between all coordinate pairs\n",
+    "    new_Q = squareform(device.interaction_coeff / pdist(new_coords) ** 6) / 4\n",
+    "    return np.linalg.norm(new_Q - Q)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "costs = []\n",
+    "np.random.seed(0)\n",
+    "x0 = np.random.random(len(Q) * 2)\n",
+    "res = minimize(\n",
+    "    evaluate_mapping,\n",
+    "    x0,\n",
+    "    args=(~np.eye(Q.shape[0], dtype=bool) * Q, device),\n",
+    "    method=\"Nelder-Mead\",\n",
+    "    tol=1e-6,\n",
+    "    options={\"maxiter\": 200000, \"maxfev\": None},\n",
+    ")\n",
+    "coords = np.reshape(res.x, (len(Q), 2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can then build the `Register` from the obtained coordinates:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qubits = {f\"q{i}\": coord for (i, coord) in enumerate(coords)}\n",
+    "reg = pulser.Register(qubits)\n",
+    "reg.draw(\n",
+    "    blockade_radius=device.rydberg_blockade_radius(1.0),\n",
+    "    draw_graph=True,\n",
+    "    draw_half_radius=True,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's check that this register matches with the device's constraints by initializing the `Sequence`: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sequence = pulser.Sequence(reg, device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "**Warning**\n",
+    "\n",
+    "This optimizer approach will not work for all the graphs. You can change the optimizer to improve it but it is a simple heuristic and should not be considered as an universal encoder. \n",
+    "\n",
+    "</div>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Pick the channels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 1 Rydberg.Global channel"
```

Comment:
This looks odd as a title. If you want to keep this structure, maybe just omitting the "1" would be better?